decidim-proposals/config/locales/ca.yml

```diff
@@ -28,6 +29,10 @@ ca:
             proposal_answering_enabled: Resposta oficial a propostes activades
             proposal_edit_before_minutes: Les propostes poden ser editades pels autors abans que passin aquest nombre de minuts
             proposal_limit: Límit de propostes per usuari
+            proposal_wizard_step_1_help_text: Text d'ajuda del primer de l'assistent de creació de la proposta
```

Comment:
We shouldn't add other locales than `en` here. All the translations are managed at Crowdin.